RELEASE-NOTES.txt

```diff
@@ -1,12 +1,17 @@
 18.8
 -----
 
-
 18.7
 -----
 * [*] Comment Reply: updated UI. [#17443, #17445]
 * [***] Two-step Authentication notifications now require an unlocked device to approve or deny them.
 * [***] Site Comments: Updated comment details with a fresh new look and capability to display rich contents. [#17466]
+* [**] Block editor: Image block: Add ability to quickly link images to Media Files and Attachment Pages [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3971]
+* [**] Block editor: Fixed a crash that could occur when copying lists from Microsoft Word. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4174]
```

Comment:
Yeah I'm not sure how @mokagio usually handles this.

In WPAndroid I've never added a `RELEASE-NOTES.txt` entry for hotfixes like he did for `18.6.1` (because those release notes don't go thru the usual process for hotfixes anyway given their timely nature), so we don't have the issue as we don't have entries for `18.6.1` anyway.

In WPiOS Gio started to do this for hotfixes which has its justifications: I actually like it given that `RELEASE-NOTES.txt` is also for us devs to keep track of when each feature/fix was shipped). But then this raises the question as to how we should cherry-pick the release notes for those cases, as it adds extra churn (both release notes will always be present in that case, once coming from the `release/18.6.1` hotfix branch since we started to add sections for those in WPiOS, and another copy coming from the `develop` branch that the `release/18.7` is cut from, before the hotfix branch gets merged later…).
So that means that when we merge back the `release/18.6.1` hotfix branch into `release/18.7` current release to resync everything, we'd have to add an extra commit to clean up the `RELEASE-NOTES.txt` file as well to remove the duplicates… not ideal either.

I'm ok either way (to clean them up and remove those 2 entries from that `REALEASE-NOTES.txt` file here as you suggest, or to keep them to enforce the idea that they were cherry-picked in both hotfix and final?)… but I'll let @mokagio decide on this one given how he handles hotfix release notes in general.

Reply:
I'm ok to mention it with this release so folks know it's been fixed (because the note wouldn't be translated for folks in 18.6.1, so they may not know).

Thanks @fluiddot!!

Reply:
Oh we're not talking about removing those from the 18.7 release notes in `AppStoreStrings.po` itself, we all agree the entries should still go into the 18.7 release notes that will be translated and displayed to end users on AppStore 🙂

The question was more about should we remove the duplicate entry in the developer-facing `RELEASE-NOTES.txt` (even if we keep it in the `.po` files) 🤷 

That's not critical either way given that's only the internal notes file (which is why I was inclined to let @mokagio do as he prefers)

Reply:
I've decided to remove the "duplicated" entry. More on the rationale here: [`b2ae4fc` (#17483)](https://github.com/wordpress-mobile/WordPress-iOS/pull/17483/commits/b2ae4fcfce7f0d81c82a7dc06f9f2f80ac7474f8)

If this was a library, I would have pushed back, but I think in this case simplicity trumps pedantry.